### ISSUE TITLE
New release?

### ISSUE THREAD

**Issue by eskimor** [paritytech/scale-info#201]
Can we have a release of scale-info please? Would be needed to be able to merge a polkadot-sdk PR. Thank you!

**Comment by niklasad1** [comment]
Yes, can you open a release PR to update the changelog and version number? 

Looks like only is https://github.com/paritytech/scale-info/pull/200 is relevant.

**Comment by niklasad1** [comment]
Thanks, released now
